examples/js/effects/VREffect.js

```diff
@@ -141,17 +134,35 @@ THREE.VREffect = function ( renderer, onError ) {
 				eyeWidth = eyeParamsL.renderWidth;
 				eyeHeight = eyeParamsL.renderHeight;
 
+				if ( vrDisplay.getLayers ) {
+
+					var layers = vrDisplay.getLayers();
+					if (layers.length) {
+
+						leftBounds = layers[0].leftBounds || [ 0.0, 0.0, 0.5, 1.0 ];
```

Comment:
`leftBounds` and `rightBounds` are already `[ 0.0, 0.0, 0.5, 1.0 ]` and `[ 0.5, 0.0, 0.5, 1.0 ]`.

How about just doing:

```js
if ( layers[ 0 ] ) {
    leftBounds = layers[ 0 ].leftBounds;
    rightBounds = layers[ 0 ].rightBounds;
}
```